testsuite/features/secondary/proxy_container_retail_mass_import.feature

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2018-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped:
+# * if there is no proxy ($proxy is nil)
+# * if there is no private network ($private_net is nil)
+# * if there is no PXE boot minion ($pxeboot_mac is nil)
+
+@containerized_server
+@skip_if_github_validation
+@proxy
+@private_net
+@pxeboot_minion
+@scope_retail
+Feature: Mass import of Retail terminals behind a containerized proxy
+  In order to manage my terminals in a Retail context
+  As the system administrator
+  I perform a mass import of several virtual terminals and one real minion
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Mass import of terminals
+    When I prepare the retail configuration file on server
+    And I import the retail configuration using retail_yaml command
+    And I am on the Systems page
+    Then I should see the terminals imported from the configuration file
+
+  # This bug prevents using mgr-bootstrap command:
+  #   https://bugzilla.suse.com/show_bug.cgi?id=1220864
+  #
+  # Scenario: Bootstrap the PXE boot minion
```

Comment:
I prefer to add a `@skip` tag on top of the scenarios that we don't want to run.
Instead of commenting the whole scenario. It implies less code lines changed, and it make obvious in the test report that we are skipping that part.

Reply:
ok.

BTW if the proxy is of Foreign type, the scanario before does not work either.

Meaning that, in this feature, unless they start fixing, nothing works :trollface: .

Reply:
Done